diff -aurN gst-plugins-bad-0.10.23.org/configure.ac gst-plugins-bad-0.10.23/configure.ac
--- gst-plugins-bad-0.10.23.org/configure.ac	2012-07-02 09:39:08.273430247 +0200
+++ gst-plugins-bad-0.10.23/configure.ac	2012-07-02 09:47:42.614001227 +0200
@@ -1300,11 +1300,11 @@
 ])
 
 dnl *** musicbrainz ***
-translit(dnm, m, l) AM_CONDITIONAL(USE_MUSICBRAINZ, true)
-AG_GST_CHECK_FEATURE(MUSICBRAINZ, [musicbrainz tag generation], musicbrainz, [
-  PKG_CHECK_MODULES(MUSICBRAINZ, libmusicbrainz >= 2.1.0, HAVE_MUSICBRAINZ="yes", HAVE_MUSICBRAINZ="no")
-  AC_SUBST(MUSICBRAINZ_CFLAGS)
-  AC_SUBST(MUSICBRAINZ_LIBS)
+translit(dnm, m, l) AM_CONDITIONAL(USE_MUSICBRAINZ5, true)
+AG_GST_CHECK_FEATURE(MUSICBRAINZ5, [musicbrainz tag generation], musicbrainz5, [
+  PKG_CHECK_MODULES(MUSICBRAINZ5, libmusicbrainz5 >= 5.0.0, HAVE_MUSICBRAINZ5="yes", HAVE_MUSICBRAINZ5="no")
+  AC_SUBST(MUSICBRAINZ5_CFLAGS)
+  AC_SUBST(MUSICBRAINZ5_LIBS)
 ])
 
 dnl *** MythTV ***
@@ -1810,7 +1810,7 @@
 AM_CONDITIONAL(USE_MPEG2ENC, false)
 AM_CONDITIONAL(USE_MPLEX, false)
 AM_CONDITIONAL(USE_MUSEPACK, false)
-AM_CONDITIONAL(USE_MUSICBRAINZ, false)
+AM_CONDITIONAL(USE_MUSICBRAINZ5, false)
 AM_CONDITIONAL(USE_MYTHTV, false)
 AM_CONDITIONAL(USE_NAS, false)
 AM_CONDITIONAL(USE_NEON, false)
diff -aurN gst-plugins-bad-0.10.23.org/ext/Makefile.am gst-plugins-bad-0.10.23/ext/Makefile.am
--- gst-plugins-bad-0.10.23.org/ext/Makefile.am	2012-07-02 09:39:08.256430460 +0200
+++ gst-plugins-bad-0.10.23/ext/Makefile.am	2012-07-02 09:50:18.526050427 +0200
@@ -220,10 +220,10 @@
 MUSEPACK_DIR=
 endif
 
-if USE_MUSICBRAINZ
-MUSICBRAINZ_DIR=musicbrainz
+if USE_MUSICBRAINZ5
+MUSICBRAINZ5_DIR=musicbrainz
 else
-MUSICBRAINZ_DIR=
+MUSICBRAINZ5_DIR=
 endif
 
 if USE_MYTHTV
